plugin/fzf.vim

```diff
@@ -140,7 +140,8 @@ try
   else
     let prefix = ''
   endif
-  let tmux = !has('nvim') && s:tmux_enabled() && s:splittable(dict)
+  let forcetmux = get(g:, 'fzf_force_tmux', 0)
+  let tmux = (!has('nvim') && s:tmux_enabled() && s:splittable(dict)) || forcetmux
```

Comment:
The option shouldn't "force". It should still check `s:tmux_enabled()` and fallback to `:term` if not running in tmux.

So a better name for the option might be `g:fzf_prefer_tmux`.